src/Compilers/CSharp/Test/Emit/PDB/CSharpDeterministicBuildCompilationTests.cs

```diff
@@ -235,7 +235,7 @@ private static IEnumerable<CSharpCompilationOptions> GetCompilationOptions()
                 delaySign: null,
                 platform: Platform.AnyCpu,
                 generalDiagnosticOption: ReportDiagnostic.Default,
-                warningLevel: 4,
+                warningLevel: 5,
```

Comment:
Should this be `Diagnostic.MaxWarningLevel`?

Reply:
@RikkiGibson Per https://github.com/dotnet/roslyn/pull/47077#discussion_r475815236, it doesn't matter.

I think we can even get it back to 4 and not touch it if it doesn't matter.

Reply:
Sounds good!

Reply:
```suggestion
                warningLevel: 4,
```